com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -1831,7 +1831,14 @@ private void HandleRawTransportPoll(NetworkEvent networkEvent, ulong clientId, A
                     // or, if we are the server, so we got everything from that client.
                     MessagingSystem.ProcessIncomingMessageQueue();
 
-                    OnClientDisconnectCallback?.Invoke(clientId);
+                    try
+                    {
+                        OnClientDisconnectCallback?.Invoke(clientId);
+                    }
+                    catch (Exception exception)
+                    {
+                        Debug.LogWarning($"Exception was thrown in OnClientDisconnectCallback: {exception}");
```

Comment:
question: should this be a Warning, Error or Exception log? what's your stance on it?